src/Umbraco.Web.BackOffice/Filters/PrefixlessBodyModelValidatorAttribute.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Umbraco.Web.BackOffice.Filters
+{
+    public class PrefixlessBodyModelValidatorAttribute : TypeFilterAttribute
+    {
+        public PrefixlessBodyModelValidatorAttribute() : base(typeof(PrefixlessBodyModelValidatorFilter))
+        {
+        }
+
+        private class PrefixlessBodyModelValidatorFilter : IActionFilter
```

Comment:
Would like to understand more about your comment: "As far as I can see there is not good why to do this anymore, unless you change it for all controllers. So this implementation is more like a hack." Maybe we need some TODO notes in here for what you tried, what might not work now, etc... and maybe a follow up task?

Reply:
Added todo